ghost/core/test/e2e-api/admin/members.test.js

```diff
@@ -2072,7 +2072,8 @@ describe('Members API', function () {
                 email: 'member1@test.com'
             });
 
-        nock('https://api.stripe.com:443')
+        const getNock = nock('https://api.stripe.com:443')
+            .persist()
```

Comment:
This getter is called multiple times, but this wasn't caught because we werent awaiting events I think